aggregator/src/app/BundleTable.ts

```diff
@@ -35,6 +36,7 @@ type Row = {
   bundle: Bundle;
   eligibleAfter: BigNumber;
   nextEligibilityDelay: BigNumber;
+  submitError: string;
```

Comment:
```suggestion
  submitError?: string;
```

Note that `RawRow` reflects what we get from the database, and therefore needs to use `null` (I think). This means adding the appropriate translation in `toRawRow`, `fromRawRow` (which TypeScript will point out).